### PR TITLE
fix: authenticating and downloading

### PR DESCRIPTION
**Pull request by arjendevos**
This isn't the complete solution but I need to add a few headers but more importantly, i needed some ssl client to get it working. I used Proxyman and enabled SSL proxying on laracasts.com and vimeo.com and it worked.

I'm not familiar with GuzzleHTTP, so tbh i've no idea why this is the case.

### CODE REVIEW

File: de0c3a72-0ef1-405c-aedf-f05fc15c651f.m4v
Comment (carlosflorencio) [inline de0c3a72-0ef1-405c-aedf-f05fc15c651f.m4v:1]: Hey @arjendevos, is this file a mistake?

**Comment by mouson** [comment]
It is not work for me. I got error message `ERROR: Client error response [url] https://laracasts.com/browse/all [status code] 403 [reason phrase] Forbidden`

File: de0c3a72-0ef1-405c-aedf-f05fc15c651f.m4v
Comment (arjendevos) [reply de0c3a72-0ef1-405c-aedf-f05fc15c651f.m4v:1]: Yes, removed it + added to git ignore

**Comment by arjendevos** [comment]
@carlosflorencio I think for it to work without an external SSL proxyman, we need to download certificates? It works in Go without proxyman. I think GuzzleHTTP does something in the background but i'm not familiar with it so i can't pinpoint why and what.

**Comment by mouson** [comment]
Thanks, it works for me now!

**Comment by suditugeorge** [comment]
> Thanks, it works for me now!

How did you make it work ? I have tried and at least for me it does not work

**Comment by mouson** [comment]
> > Thanks, it works for me now!
> 
> How did you make it work ? I have tried and at least for me it does not work

Did you have any error message ?

**Comment by AmirRezaM75** [comment]
I guess we no longer have this issue. or at least not everyone have this issue

**Pull request closed without merging**